tools/clang/test/CodeGenSPIRV/binary-op.assign.hlsl

```diff
@@ -0,0 +1,23 @@
+// Run: %dxc -T ps_6_0 -E main
+
+// TODO: assignment for composite types
+
+void main() {
+    int a, b, c;
+
+// CHECK: %[[b0:[0-9]+]] = OpLoad %int %b
```

Comment:
Also, just stylisically consider including the % in the regex pattern.
So you'd have rules like:

```
CHECK: [[b0:%\d+]] = OpLoad %int %b
CHECK-NEXT: OpStore %a [[b0]]
```

Up to you.   For me, this indicates that `b0` is the identifier, not just the number of the identifier.